src/components/HURUmap/CategoryHeader/index.js

```diff
@@ -0,0 +1,42 @@
+import { Typography, SvgIcon } from "@material-ui/core";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+const CategoryHeader = ({ title, description, icon }) => {
+  const classes = useStyles();
+
+  if (!title) {
+    return null;
+  }
+  return (
+    <div className={classes.root}>
+      <Typography variant="h3" className={classes.title}>
+        <SvgIcon
+          className={classes.svgIcon}
+          component={icon}
```

Comment:
To confirm, does this work with icon links with get from CMS? e.g. does a link like <https://cms.dev.codeforafrica.org/pesayetu/wp-content/uploads/sites/2/2021/08/Group-4658-1.svg> work?

Reply:
This does not work, an element type is needed to be passed to the SvgIcon component @kilemensi I propose switching it back to next/images and look into It in a different PR as everything else is ready in this PR

Reply:
Yeah was afraid of that @gertie-sheshe ... We'll probably need something like https://stackoverflow.com/a/52967417 but it feels like too much work for something so small. So yes, lets just render SVG using `Image` component.

Reply:
Thanks. Was looking at that exact link with @KobbyMmo yesterday too.